src/opm/input/eclipse/EclipseState/Tables/SimpleTable.cpp

```diff
@@ -16,34 +16,41 @@
   You should have received a copy of the GNU General Public License
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include <utility>
-#include <iostream>
 
 #include <opm/input/eclipse/EclipseState/Tables/SimpleTable.hpp>
+
 #include <opm/input/eclipse/EclipseState/Tables/TableSchema.hpp>
+
 #include <opm/input/eclipse/Deck/DeckItem.hpp>
 
+#include <stdexcept>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <iostream>
+
 #include <fmt/format.h>
 
 namespace Opm {
 
-    SimpleTable::SimpleTable( TableSchema schema, const std::string& tableName, const DeckItem& deckItem,
-                              const int tableID) :
-        m_schema( std::move( schema ) ),
-        m_jfunc (false)
+    SimpleTable::SimpleTable(TableSchema        schema,
```

Comment:
since we're basically touching every line of this file anyways, maybe drop indent for namespace?